.github/workflows/ci.yml

```diff
@@ -43,6 +43,14 @@ jobs:
         if: github.event_name == 'pull_request'
         run: echo "BRANCH_NAME=$(echo $GITHUB_HEAD_REF)" >> $GITHUB_ENV
 
+      # extract repository name and owner
+      - name: Extract repo name
+        run: echo "REPO_NAME=$(echo ${{ github.repository }}.git)" >> $GITHUB_ENV
```

Comment:
Should we modify the variable name to include an `owner` there?

Reply:
To `REPO_NAME_AND_ONWER`? We can do that but I wonder if it's not too verbose? Wyt?

Reply:
Actually, I think the `REPO_NAME` is fine. Let's leave it